inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -1008,90 +1009,54 @@ void MKLDNNConvolutionNode::prepareParams() {
             auto reordItpd = reorderConvDesc->createPrimitiveDescriptorIterator(getEngine(), *pAttrLocal);
             if (static_cast<bool>(reordItpd)) {
                 auto prim_desc = convolution_forward::primitive_desc(reordItpd.get());
-                execPtr = std::make_shared<ConvolutionExecutor>(prim_desc, srcMemPtr, wghMemPtr, dstMemPtr, *pAttrLocal,
-                                                                binaryPostOpsArgs, getEngine(), biasMemPtr);
+                execPtr = std::make_shared<ConvolutionExecutor>(prim_desc, srcMemPtr, wghMemPtr, dstMemPtr, getEngine());
+                break;
             }
         }
     }
-    IE_THROW() << "Primitive descriptor was not found for node " << getName() << ".";
-}
+    if (execPtr) {
+        primArgs[DNNL_ARG_SRC] = getParentEdgesAtPort(0)[0]->getMemoryPtr()->GetPrimitive();
+        primArgs[DNNL_ARG_WEIGHTS] = getWeights();
+        primArgs[DNNL_ARG_DST] = getChildEdgesAtPort(0)[0]->getMemoryPtr()->GetPrimitive();
```

Comment:
It looks like we do not need to call `getParentEdgesAtPort(0)[0]->getMemoryPtr()` once again, we already have all the necessary memory pointer.